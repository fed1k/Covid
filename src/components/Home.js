import React from 'react';
import Search from './searchBar';
import Countries from './Countries';

const Home = () => (
  <div>
    <Search />
    <Countries />
  </div>
);

export default Home;
