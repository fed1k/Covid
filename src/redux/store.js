import { configureStore } from '@reduxjs/toolkit';
import myReducer from './reducer';

const store = configureStore({
  reducer: {
    country: myReducer,
  },
});

export default store;
