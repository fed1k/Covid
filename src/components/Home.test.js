import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Home from './Home';
import store from '../redux/store';

test('should match snapshot', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Home />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
