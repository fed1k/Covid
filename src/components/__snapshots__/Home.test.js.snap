// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <form>
    <svg
      className="search-icon"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 16 16"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"
      />
    </svg>
    <input
      id="search"
      name="search"
      onChange={[Function]}
      placeholder="Search..."
      type="search"
    />
  </form>
  <div
    className="cards-main-container"
  >
    <img
      alt="loading"
      src="Loading_icon.gif"
    />
  </div>
</div>
`;
